connectors/sources/notion.py

```diff
@@ -443,3 +555,53 @@ async def get_docs(self, filtering=None):
                     block_id=block_id
                 ):
                     yield self._format_doc(comment), None
+
+    async def get_docs(self, filtering=None):
```

Comment:
Please add some logging so that it's clear which flow we're going down, and what queries are being executed against the notion client.